pandas/conftest.py

```diff
@@ -250,7 +246,10 @@ def pytest_collection_modifyitems(items, config) -> None:
     )
 
 
-@pytest.fixture
+# ----------------------------------------------------------------
+# Autouse fixtures
+# ----------------------------------------------------------------
+@pytest.fixture(autouse=True)
```

Comment:
Could you add a reference to the pytest issue you opened? (Still hoping we don't have to use an `autouse` fixture one day)

Reply:
Done